src/components/FormModal/AlertModal.js

```diff
@@ -1,4 +1,13 @@
 import "./AlertModal.css";
+import PropTypes from "prop-types";
+
+AlertModal.propTypes = {
+  conflictName: PropTypes.string,
+  clickedWeekdayName: PropTypes.string,
+  clickedTimeOfTheDay: PropTypes.string,
+  onProductSwapClicked: PropTypes.func,
+  onCancelAlertModal: PropTypes.func,
```

Comment:
I would only leave out `.required` on a prop type if the component or function works without the prop or parameter
It would look like this:
`onProductSwapClicked: PropTypes.func.required`